### ISSUE TITLE
Any way to create SKBitmap to use in SkiaSharp?

### ISSUE THREAD

**Issue by idchlife** [Redth/ZXing.Net.Mobile#882]
Hello! Thank you very much for all of your work for bringing this nice library to us!

Is it possible to create SKBitmap with this library? Trying to insert barcode into an image, but it seems no way I could create valid byte[] for using in SKBitmap.Decode (always returns null), I even tried using ImageSharp to create byte array - no luck.

I discovered that SKBitmap bindings exists in ZXing.Net library, but it does work only on iOS devices, Android devices throw errors about zxing library references all the time :(

**Comment by micjahn** [comment]
Would you post some more details about the reference errors on android, please?

**Comment by idchlife** [comment]
@micjahn sorry, I gone past this stage some time ago :( It was something like

"Cannot resolve reference zxing from MyMainApp, please install zxing refernce in MyMainAppAndroid or remove reference to MyMainApp"

zxing part I remember clearly, because it was a surprise for me that it tries to load zxing library and not zxing.net

Anyway, I resolved my problem by manually copy-pasting this: https://github.com/micjahn/ZXing.Net/blob/master/Source/Bindings/ZXing.SkiaSharp/Renderer/SKBitmapRenderer.cs

into the project and using it there.

Basically I think this class may be useful addition to ZXing.Net.Mobile since SkiaSharp is library widely used in Xamarin ecosystem. I was surprised to see this renderer inside ZXing.Net and not inside ZXing.Net.Mobile

**Comment by micjahn** [comment]
No surprise for me. The name of the assembly is zxing, only the project name is ZXing.Net. Without further details of the reference error I can't help here. Support for SkiaSharp inside ZXing.Net.Mobile has to be evaluated by @Redth
